PayPalCheckout/src/main/java/com/paypal/android/checkout/error/PayPalError.kt

```diff
@@ -0,0 +1,23 @@
+package com.paypal.android.checkout.error
+
+import com.paypal.android.core.CoreSDKError
+
+internal object PayPalError {
+
+    // 0. An unknown error occurred.
+    val unknownError = CoreSDKError(
+        code = Code.UNKNOWN.ordinal,
+        errorDescription = "An unknown error occurred. Contact developer.paypal.com/support."
+    )
+
+    // 1. Result did not contain the expected data.
+    val malformedResultError = CoreSDKError(
+        code = Code.MALFORMED_RESULT.ordinal,
+        errorDescription = "Result did not contain the expected data. Payer ID or Order ID is null."
+    )
+}
+
+internal enum class Code {
```

Comment:
These codes are not accessible to merchants - we return the error as a `CoreSDKError` with a description, but the code will just be an Int to the merchant. I think this is the same in iOS. Does this make sense? In BT we just return a description in `BraintreeException`. Do we need these codes? Moving them all to the CoreSDKError would make them visible to the merchant, but wouldn't support modularity.

Reply:
There's also this if we need it [Library Annotation](https://developer.android.com/studio/write/annotations#libraries) 🙌